shell/store/prefs.js

```diff
@@ -91,7 +91,13 @@ export const TIME_FORMAT = create('time-format', 'h:mm:ss a', {
 });
 
 export const TIME_ZONE = create('time-zone', 'local');
+// DEV will be deprecated on v2.7.0, but is needed so that we can grab the value for the new settings that derived from it
+// such as: VIEW_IN_API, ALL_NAMESPACES, NO_LOCALE_SHORTCUT, THEME_SHORTCUT
 export const DEV = create('dev', false, { parseJSON });
+export const VIEW_IN_API = create('view-in-api', mapPref(DEV) || false, { parseJSON });
+export const ALL_NAMESPACES = create('all-namespaces', mapPref(DEV) || false, { parseJSON });
+export const NO_LOCALE_SHORTCUT = create('no-locale-shortcut', mapPref(DEV) || false, { parseJSON });
+export const THEME_SHORTCUT = create('theme-shortcut', mapPref(DEV) || false, { parseJSON });
```

Comment:
I tested coming from an instance with the original "Enabled Developer Tools & Features" or `DEV` checkbox both enabled and disabled.
When the checkbox was disabled originally, all of the new checkboxes would default to `true`. It seems the `mapPref(DEV) || false` is returning `true` regardless of the initial setting.

Reply:
Oh my, this was so wrong... What was I thinking? I didn't even test this properly... Sorry about that 🙏 

On my latest commit I've added the correct logic on the `prefs` store to inherit from `DEV`

Reply:
No worries! Looks like your changes fixed the problem :sunglasses: